estate/views/estate_property_views.xml

```diff
@@ -127,7 +121,7 @@
         <field name="model">property</field>
         <field name="arch" type="xml">
             <search>
-                <filter string="Available" name="available" domain="[('state', '=', 'available')]"/>
+                <filter string="Available" name="available" domain="[('state', '=', ['new', 'offer_received'])]"/>
```

Comment:
This should be using the equal operator

and we even have a warning about this

![Screenshot 2023-05-05 at 6 02 00 p m](https://user-images.githubusercontent.com/47955967/236586828-81c2cbb4-b7c1-4537-ab0c-03dc917191a1.png)